aidial_adapter_openai/gpt.py

```diff
@@ -49,27 +50,30 @@ async def gpt_chat_completion(
         )
 
     client = chat_completions_parser.parse(upstream_endpoint).get_client(
-        {**creds, "api_version": api_version, "timeout": DEFAULT_TIMEOUT}
+        {**creds, "api_version": api_version}
     )
 
     response: AsyncStream[ChatCompletionChunk] | ChatCompletion = (
         await call_with_extra_body(client.chat.completions.create, data)
     )
 
     if isinstance(response, AsyncStream):
+
+        def to_dict(chunk: ChatCompletionChunk) -> dict:
```

Comment:
I you want to use this `to_dict` function to save space, you should consider to move it into `utils`, and re-use it in `databricks` and `mistral` modules, they both have this part


```python
                map_stream(
                    debug_print,
                    map_stream(lambda chunk: chunk.to_dict(), response),
                )
```

It makes sense to do this also perfomance-wise:

Each time we call this `map_stream` twice, we traverse all elements just to make this check for `debug_print`

Reply:
Fixed